game/src/engine/audio/AudioService.cpp

```diff
@@ -22,75 +25,359 @@ const ALCchar* kDefaultDevice = nullptr;
 ALCint kContextAttributes = 0;
 
 // audio directories
-// note: since diagetic sound (3d sound) can only be in mono
-// we'll want all sfx to be mono too (music is fine in stereo)
 const std::string kSfxDirectory = "resources/audio/sfx/";
 const std::string kMusicDirectory = "resources/audio/music/";
```

Comment:
Usually it's a good idea to add `static` to variables that are meant to only exist in one source file/compilation unit. reason being, just by looking at these vars here, there's no way to tell if they're only meant for this file or if there's a header file somewhere defining like:
```c++
extern ALCint kContextAttributes
```
If a header _did_ exist like this, these vars would then refer to the thing in the header.

tl;dr: being explicit usually makes it easier to determine what stuff means when looking at it later :P